tools/scripts-config-react-webpack/config/webpack.config.js

```diff
@@ -394,10 +394,9 @@ module.exports = ({ getUserConfig, mode }) => {
 				},
 				static: {
 					directory: path.join(process.cwd(), 'dist'),
-					watch: true,
```

Comment:
need to set it to false. default value is true: https://webpack.js.org/configuration/dev-server/#watch

Reply:
I prefer remove it, it was a mistake, static is to add a new folder to watch, we do not need this